examples/using-js-search/README.md

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Should these search example files be here? Seems like those are from another example.